rasa/nlu/extractors/crf_entity_extractor.py

```diff
@@ -22,11 +30,19 @@
     from spacy.tokens import Doc
 
 
+class CRFToken(NamedTuple):
+    text: Text
+    tag: Text
+    entity: Text
+    pattern: Dict[Text, Any]
+    word_embedding: np.ndarray
```

Comment:
should we use `namedtuple` directly?
`CRFToken = namedtuple("CRFToken", ("text", "tag", "entity", "pattern", "word_embedding"))`

Reply:
I think we discussed this at some point with @tmbo and decided on later

Reply:
The benefit of `class CRFToken(NamedTuple)` is that you can define types.